<launch>
<include file="$(find turtlebot_bringup)/launch/minimal.launch" />

 <arg name="initial_pose_x" default="0.0"/> 
 <arg name="initial_pose_y" default="0.0"/> 
 <arg name="initial_pose_a" default="-2.0"/>
  <include file="$(find deminer)/launch/custom_amcl.launch" >
 <arg name="initial_pose_x" value="$(arg initial_pose_x)"/>
 <arg name="initial_pose_y" value="$(arg initial_pose_y)"/>
 <arg name="initial_pose_a" value="$(arg initial_pose_a)"/>
  </include>

        <!-- This is pretty much for testing purposes -->
        <!-- <include file="$(find turtlebot_gazebo)/launch/turtlebot_world.launch" />
             <include file="$(find turtlebot_gazebo)/launch/gmapping_demo.launch" />
             <include file="$(find turtlebot_navigation)/launch/includes/move_base.launch.xml" /> -->

<node name="playing_sound_node" pkg="sound_play" type="soundplay_node.py" output="screen"/>

<node name="rviz" pkg="rviz" type="rviz" args="-d $(find deminer)/config/coordrun.rviz"/>

<node name= "boundary_selection" type="boundary_selection" pkg="deminer" output="screen" launch-prefix="gnome-terminal --command"/>

</launch>